### ISSUE TITLE
EMFILE: too many open files

### ISSUE THREAD

**Issue by rlyle** [serverless/serverless#10706]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

I'm trying to run sls package to create a package from our build directory and getting a "EMFILE: too many open files' error when trying to create a package (please note, this is for a package that is deploying file from the bitbucket pipelines, I'm just trying to create a package locally on my windows box to test deploying a built package to our other environments).

### Service configuration (serverless.yml) content

```yaml
service: services
provider:
  tracing:
    apiGateway: true
    lambda: true
  name: aws
  runtime: nodejs12.x
  endpointType: REGIONAL
  region: ${file(./config/${opt:stage}.json):aws.region}
  memorySize: 3008
  timeout: 300
  logRetentionInDays: 7
  environment:
    STAGE: ${opt:stage}
    NODE_ENV: ${opt:stage}
    ENABLE_DB_AUTO_INDEX: "true"
    PATH_PREFIX: ${env:PATH_PREFIX, ''}
    HG_CLIENT_ID: ${env:HG_CLIENT_ID, ''}
    HG_CLIENT_SECRET: ${env:HG_CLIENT_SECRET, ''}
    HG_PROVIDER_ID: ${env:HG_PROVIDER_ID, ''}
    HG_PROVIDER_SECRET: ${env:HG_PROVIDER_SECRET, ''}
    HG_BOX_CLIENT_ID: ${env:HG_BOX_CLIENT_ID, ''}
    HG_BOX_CLIENT_SECRET: ${env:HG_BOX_CLIENT_SECRET, ''}
    HG_BOX_PROVIDER_ID: ${env:HG_BOX_PROVIDER_ID, ''}
    HG_BOX_PROVIDER_SECRET: ${env:HG_BOX_PROVIDER_SECRET, ''}
    KNO2_CLIENT_ID: ${env:KNO2_CLIENT_ID, ''}
    KNO2_KEY: ${env:KNO2_KEY, ''}
    KNO2_SECRET: ${env:KNO2_SECRET, ''}
    KNO2_QA_CLIENT_ID: ${env:KNO2_QA_CLIENT_ID, ''}
    KNO2_QA_KEY: ${env:KNO2_QA_KEY, ''}
    KNO2_QA_SECRET: ${env:KNO2_QA_SECRET, ''}
    ENABLE_DB_DEBUGGING: "false"
    DB_POOLSIZE: "2"
    HG_QA_CLIENT_SECRET: ${env:HG_QA_CLIENT_SECRET, ''}
    HG_QA_CLIENT_ID: ${env:HG_QA_CLIENT_ID, ''}
    JIRA_USER: ${env:JIRA_USER, ''}
    JIRA_API_TOKEN: ${env:JIRA_API_TOKEN, ''}
    TWILIO_SID: ${env:TWILIO_SID, ''}
    TWILIO_SID_TOKEN: ${env:TWILIO_SID_TOKEN, ''}
    TWILIO_AUTH_TOKEN: ${env:TWILIO_AUTH_TOKEN, ''}
    TWILIO_SECRET: ${env:TWILIO_SECRET, ''}
    MAXMD_APRICITYEMAIL: ${env:MAXMD_APRICITYEMAIL, ''}
    MAXMD_APRICITYPW: ${env:MAXMD_APRICITYPW, ''}
    ZENDESK_USERNAME: ${env:ZENDESK_USERNAME, ''}
    ZENDESK_TOKEN: ${env:ZENDESK_TOKEN, ''}
    JIRA_FB_USERNAME: ${env:JIRA_FB_USERNAME, ''}
    JIRA_FB_PASSWORD: ${env:JIRA_FB_PASSWORD, ''}
    HG_QA_PROVIDER_SECRET: ${env:HG_QA_PROVIDER_SECRET, ''}
    HG_QA_PROVIDER_ID: ${env:HG_QA_PROVIDER_ID, ''}
    APRICITY_OID: ${env:APRICITY_OID, ''}
    BITBUCKET_USERNAME: ${env:BITBUCKET_USERNAME, ''}
    BITBUCKET_PASSWORD: ${env:BITBUCKET_PASSWORD, ''}
  iamRoleStatements:
    - Effect: Allow
      Action:
        - dynamodb:PutItem
        - dynamodb:GetItem
        - xray:PutTraceSegments
        - xray:PutTelemetryRecords
        - ec2:CreateNetworkInterface
        - ec2:DescribeNetworkInterfaces
        - ec2:DetachNetworkInterface
        - ec2:DeleteNetworkInterface
        - lambda:*
        - sns:*
        - s3:*
        - lambda:InvokeFunction
        - forecast:*
        - iam:PassRole
        - sqs:*
        - logs:CreateLogGroup
        - logs:CreateLogStream
        - logs:PutLogEvents
        - batch:DescribeJobs
        - cognito-idp:*
        - SNS:OptInPhoneNumber
        - translate:TranslateText
        - cloudfront:ListDistributions
        - cloudfront:CreateInvalidation
        - athena:*
        - glue:*
        - SNS:Publish
        - ses:SendRawEmail
        - route53:ChangeResourceRecordSets
        - cloudfront:GetDistribution
        - cloudfront:UpdateDistribution
      Resource:
        - "*"
    - Effect: Allow
      Action:
        - s3:ListBucket
        - s3:GetObject
        - s3:PutObject
        - s3:PutObjectAcl
        - s3:GetObjectTagging
        - s3:PutObjectTagging
      Resource:
        - arn:aws:s3:::education.apricity-health.com
        - arn:aws:s3:::education.apricity-health.com/*
  vpc: ${file(./config/${opt:stage}.json):aws.vpc}
  batch:
    Type: EC2
    BidPercentage: 100
    SecurityGroupIds: ${file(./config/${opt:stage}.json):aws.vpc.securityGroupIds}
    Subnets: ${file(./config/${opt:stage}.json):aws.vpc.subnetIds}
    InstanceTypes:
      - c5.large
    MinvCpus: 0
    MaxvCpus: 8
functions:
  handleProxyRequest:
    handler: index.handleProxyRequest
    events:
      - http:
          path: /{proxy+}
          method: any
          authorizer:
            arn: ${file(./config/${opt:stage}.json):aws.user_pool_arn}
          cors:
            origins:
              - "*"
  conversation:
    handler: index.conversation
  detect:
    handler: index.detect
  getPatients:
    handler: index.getPatients
  getPatientData:
    handler: index.getPatientData
  diff:
    handler: index.diff
  mergePlans:
    handler: index.mergePlans
  flattenPlan:
    handler: index.flattenPlan
  notifyPatient:
    handler: index.notifyPatient
  notifyProvider:
    handler: index.notifyProvider
  saveImage:
    handler: index.saveImage
  importDirectMessages:
    handler: index.importDirectMessages
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enableImportDirectMessages}
          rate: cron(0/15 * ? * * *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  processAttachments:
    handler: index.processAttachments
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enableImportDirectMessages}
          rate: cron(0/1 * ? * * *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  processAttachment:
    timeout: 3600
    handler: index.processAttachment
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  validateSubscriptions:
    handler: index.validateSubscriptions
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enableValidateSubscriptions}
          rate: cron(0/60 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  pushFHIR:
    handler: index.pushFHIR
  subscriptionFHIR:
    handler: index.subscriptionFHIR
    events:
      - http:
          path: /apricity-cleanse/subscription/fhir
          method: post
          cors:
            origins:
              - "*"
  subscriptionFHIRmeta:
    handler: index.subscriptionFHIRmeta
    events:
      - http:
          path: /apricity-cleanse/subscription/fhir/metadata
          method: get
          cors:
            origins:
              - "*"
  importPatient360:
    timeout: 43200
    handler: index.importPatient360
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enablePatient360Import}
          rate: cron(0 6 * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  processRecordImports:
    timeout: 900
    handler: index.processRecordImports
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enablePatient360Import}
          rate: cron(0/15 * ? * * *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  importPatients:
    timeout: 14400
    handler: index.importPatients
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enableImportPatients}
          rate: cron(0 0/4 ? * * *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
  importPatient:
    timeout: 3600
    handler: index.importPatient
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  exportCleanPatients:
    handler: index.exportCleanPatients
    timeout: 900
    memorySize: 3008
    events:
      - schedule:
          rate: ${file(./config/${opt:stage}.json):exportFlaggedPatients.cron}
          enabled: ${file(./config/${opt:stage}.json):exportFlaggedPatients.enabled}
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
            queryStringParameters:
              exportFlag: "true"
              fileId: ${file(./config/${opt:stage}.json):exportFlaggedPatients.fileId}
      - schedule:
          rate: cron(0 9 ? * 2 *)
          enabled: ${file(./config/${opt:stage}.json):exportAllPatients}
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
            queryStringParameters:
              fileId: all-patients
  exportCleanPatientsAsync:
    handler: index.exportCleanPatientsAsync
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  importCleanPatients:
    handler: index.importCleanPatients
    timeout: 300
    memorySize: 3008
    events:
      - schedule:
          rate: ${file(./config/${opt:stage}.json):importFlaggedPatients.cron}
          enabled: ${file(./config/${opt:stage}.json):importFlaggedPatients.enabled}
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-cleanse
                  cognito:groups: administrators
            queryStringParameters:
              userId: ${file(./config/${opt:stage}.json):importFlaggedPatients.userId}
              fileId: ${file(./config/${opt:stage}.json):importFlaggedPatients.fileId}
              firstName: ${file(./config/${opt:stage}.json):importFlaggedPatients.firstName}
              lastName: ${file(./config/${opt:stage}.json):importFlaggedPatients.lastName}
              providerId: ${file(./config/${opt:stage}.json):synthetic.providerId}
  importCleanPatientsAsync:
    handler: index.importCleanPatientsAsync
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  saveDataIntoS3:
    handler: index.saveDataIntoS3
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  importForecastData:
    handler: index.importForecastData
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  updateInstances:
    handler: index.updateInstances
    events:
      - schedule:
          rate: cron(0/1 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: apricity-forecast
                  cognito:groups: administrators,debug
  processAlerts:
    memorySize: 3008
    handler: index.processAlerts
    events:
      - schedule:
          rate: cron(0/1 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: alerts
                  cognito:groups: administrators,debug
  register:
    handler: index.register
    events:
      - http:
          path: /authentication/register
          method: post
          cors:
            orgins:
              - "*"
  login:
    handler: index.login
    events:
      - http:
          path: /authentication/login
          method: post
          cors:
            origins:
              - "*"
  refresh:
    handler: index.refresh
    events:
      - http:
          path: /authentication/refresh
          method: post
          cors:
            origins:
              - "*"
  changePassword:
    handler: index.changePassword
    events:
      - http:
          path: /authentication/changePassword
          method: post
          cors:
            origins:
              - "*"
  resetPassword:
    handler: index.resetPassword
    events:
      - http:
          path: /authentication/resetPassword
          method: post
          cors:
            origins:
              - "*"
  confirmPassword:
    handler: index.confirmPassword
    events:
      - http:
          path: /authentication/confirmPassword
          method: post
          cors:
            origins:
              - "*"
  linkExternalAccount:
    handler: index.linkExternalAccount
    events:
      - http:
          path: /authentication/linkAccount
          method: post
          cors:
            origins:
              - "*"
  resetMFA:
    handler: index.resetMFA
    events:
      - http:
          path: /authentication/mfa/reset
          method: post
          cors:
            origins:
              - "*"
  userMigration:
    handler: index.userMigration
  getText:
    handler: index.getText
    events:
      - http:
          path: /content/text
          method: get
          request:
            parameters:
              queryStrings:
                planId: false
                textId: false
                language: false
                category: false
                version: false
                dependencies: false
          cors:
            origins:
              - "*"
  handleCognitoMessageTrigger:
    memorySize: 3008
    handler: index.handleCognitoMessageTrigger
  processSMSReply:
    handler: index.processSMSReply
  processDirectMessages:
    handler: index.processDirectMessages
    events:
      - schedule:
          enabled: ${file(./config/${opt:stage}.json):enableProcessDirectMessages}
          rate: cron(0/1 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: messaging
                  cognito:groups: administrators
  processBroadcastMessages:
    handler: index.processBroadcastMessages
    events:
      - schedule:
          rate: cron(0/5 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: messaging
                  cognito:groups: administrators
  sendBroadcastMessage:
    handler: index.sendBroadcastMessage
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  getOrgByDomain:
    handler: index.getOrgByDomain
    events:
      - http:
          path: /orgs/domain/{dns}
          method: get
          request:
            parameters:
              paths:
                dns: true
          cors:
            origins:
              - "*"
  archivePatientData:
    memorySize: 3008
    timeout: 900
    handler: index.archivePatientData
    events:
      - schedule:
          rate: cron(0 2 * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: patients
                  cognito:groups: administrators
  deletePatient:
    handler: index.deletePatient
    memorySize: 3008
    events:
      - schedule:
          rate: cron(0 6 * * ? *)
          enabled: ${file(./config/${opt:stage}.json):delete_test_patients}
          input:
            pathParameters:
              patientId: "*"
            queryStringParameters:
              firstName: Test
            requestContext:
              authorizer:
                claims:
                  sub: patients
                  cognito:groups: administrators
  deactivateReminders:
    handler: index.deactivateReminders
    timeout: 900
    events:
      - schedule:
          rate: cron(0 6 * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: patients
                  cognito:groups: administrators
  runReportSchedules:
    handler: index.runReportSchedules
    events:
      - schedule:
          rate: cron(0/5 * * * ? *)
          input:
            requestContext:
              authorizer:
                claims:
                  sub: reporting
                  cognito:groups: administrators,debug
  runReportJob:
    handler: index.runReportJob
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
  deleteReportJob:
    handler: index.deleteReportJob
    events:
      - schedule:
          rate: cron(0 * * * ? *)
          input:
            pathParameters:
              jobId: "*"
            queryStringParameters:
              days: "7"
            requestContext:
              authorizer:
                claims:
                  sub: reporting
                  cognito:groups: administrators
  exportData:
    handler: index.exportData
    timeout: 900
    events:
      - schedule:
          rate: cron(0 6 ? * 1 *)
          enabled: ${file(./config/${opt:stage}.json):data_export_enabled}
          input:
            pathParameters:
              planId: 6d16691e-b336-4d8e-be91-d80d4b547cd7
              contentId: irae
            requestContext:
              authorizer:
                claims:
                  sub: rosetta
                  cognito:groups: administrators
      - schedule:
          rate: cron(15 6 ? * 1 *)
          enabled: ${file(./config/${opt:stage}.json):data_export_enabled}
          input:
            pathParameters:
              planId: 63575a2f-2c1e-4e2f-a0e2-a908a1ae0354
              contentId: covid-19-rtw
            requestContext:
              authorizer:
                claims:
                  sub: rosetta
                  cognito:groups: administrators
      - schedule:
          rate: cron(30 6 ? * 1 *)
          enabled: ${file(./config/${opt:stage}.json):data_export_enabled}
          input:
            pathParameters:
              planId: ce1fe0d5-ac66-45e9-9dd2-d0bb31b80261
              contentId: common
            requestContext:
              authorizer:
                claims:
                  sub: rosetta
                  cognito:groups: administrators
      - schedule:
          rate: cron(45 6 ? * 1 *)
          enabled: ${file(./config/${opt:stage}.json):data_export_enabled}
          input:
            pathParameters:
              planId: 5f4897fa-f898-439b-b007-ddb386c5a581
              contentId: oncology
            requestContext:
              authorizer:
                claims:
                  sub: rosetta
                  cognito:groups: administrators
      - schedule:
          rate: cron(0 7 ? * 1 *)
          enabled: ${file(./config/${opt:stage}.json):data_export_enabled}
          input:
            pathParameters:
              planId: e93e2337-0100-4fe4-bc16-00fa635ccc98
              contentId: chemo-tox
            requestContext:
              authorizer:
                claims:
                  sub: rosetta
                  cognito:groups: administrators
  runTests:
    memorySize: 3008
    handler: index.runTests
    events:
      - schedule:
          rate: ${file(./config/${opt:stage}.json):run_tests_irae}
          enabled: ${file(./config/${opt:stage}.json):run_tests_enabled}
          input:
            pathParameters:
              planId: 6d16691e-b336-4d8e-be91-d80d4b547cd7
            requestContext:
              authorizer:
                claims:
                  sub: test
                  cognito:groups: administrators
      - schedule:
          rate: ${file(./config/${opt:stage}.json):run_tests_covid_rtw}
          enabled: ${file(./config/${opt:stage}.json):run_tests_enabled}
          input:
            pathParameters:
              planId: 63575a2f-2c1e-4e2f-a0e2-a908a1ae0354
            requestContext:
              authorizer:
                claims:
                  sub: test
                  cognito:groups: administrators
  createTestPatients:
    handler: index.createTestPatients
    events:
      - schedule:
          rate: ${file(./config/${opt:stage}.json):create_test_patients_cron}
          enabled: ${file(./config/${opt:stage}.json):create_test_patients_enabled}
          input:
            requestContext:
              authorizer:
                claims:
                  sub: test
                  cognito:groups: administrators
  simulateConversations:
    handler: index.simulateConversations
  runMigrationBatch:
    handler: index.runMigrationBatch
    timeout: 43200
    batch:
      ContainerProperties:
        Memory: 3008
        Vcpus: 1
        Environment:
          AWS_LAMBDA_FUNCTION_MEMORY_SIZE: 3008
          LOG_LAMBDA: log-stream-${opt.stage}-log
plugins:
  - serverless-domain-manager
  - serverless-log-forwarding
  - serverless-apigw-binary
  - serverless-apigwy-binary
  - serverless-plugin-tracing
  - "@rlyle1179/serverless-offline"
  - "@rlyle1179/serverless-aws-batch"

custom:
  customDomain:
    domainName: ${file(./config/${opt:stage}.json):domain}
    basePath: ${env:PATH_PREFIX, ''}api
    stage: ${opt:stage}
    createRoute53Record: true
  logForwarding:
    destinationARN: arn:aws:lambda:${self:provider.region}:${env:AWS_ACCOUNT_ID}:function:log-stream-${opt:stage}-stream
  apigwBinary:
    types:
      - "image/jpeg"
      - "image/png"
      - "application/pdf"
  serverless-offline:
    httpPort: 2999
    lambdaPort: 3999
```

### Command name and used flags

sls package --stage develop --package ../package

### Command output

```shell
$ sls package --stage develop --package ../package
Running "serverless" from node_modules

Warning: Invalid configuration encountered
  at 'functions.processAttachment': unrecognized property 'batch'
  at 'functions.processAttachment.timeout': must be <= 900
  at 'functions.importPatient360': unrecognized property 'batch'
  at 'functions.importPatient360.timeout': must be <= 900
  at 'functions.importPatients': unrecognized property 'batch'
  at 'functions.importPatients.timeout': must be <= 900
  at 'functions.importPatient': unrecognized property 'batch'
  at 'functions.importPatient.timeout': must be <= 900
  at 'functions.exportCleanPatientsAsync': unrecognized property 'batch'
  at 'functions.exportCleanPatientsAsync.timeout': must be <= 900
  at 'functions.importCleanPatientsAsync': unrecognized property 'batch'
  at 'functions.importCleanPatientsAsync.timeout': must be <= 900
  at 'functions.saveDataIntoS3': unrecognized property 'batch'
  at 'functions.saveDataIntoS3.timeout': must be <= 900
  at 'functions.importForecastData': unrecognized property 'batch'
  at 'functions.importForecastData.timeout': must be <= 900
  at 'functions.register.events.0.http.cors': unrecognized property 'orgins'
  at 'functions.getText.events.0.http.request.parameters': unrecognized property 'queryStrings'
  at 'functions.sendBroadcastMessage': unrecognized property 'batch'
  at 'functions.sendBroadcastMessage.timeout': must be <= 900
  at 'functions.runReportJob': unrecognized property 'batch'
  at 'functions.runReportJob.timeout': must be <= 900
  at 'functions.runMigrationBatch': unrecognized property 'batch'
  at 'functions.runMigrationBatch.timeout': must be <= 900
  at 'provider': unrecognized property 'batch'
  at 'provider.vpc': unrecognized property 'id'

Learn more about configuration validation here: http://slss.io/configuration-validation

Packaging services for stage develop (us-east-1)
Updating Log Forwarding Resources...
Log Forwarding Resources Updated
Environment: win32, node 14.18.3, framework 3.2.0 (local) 3.2.1v (global), plugin 6.0.0, SDK 4.3.1
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Cannot read file node_modules\npm\lib\bugs.js due to: EMFILE: too many open files, open 'F:\Work\apricity-platform\services\build\node_modules\npm\lib\bugs.js'

2 deprecations found: run 'serverless doctor' for more details

rlyle@MERCURY MINGW64 /f/Work/apricity-platform/services/build (develop)
```

### Environment information

```shell
$ sls --version
Running "serverless" from node_modules
Framework Core: 3.2.0 (local) 3.2.1v (global)
Plugin: 6.0.0
SDK: 4.3.1
```

**Comment by pgrzesik** [comment]
Hello @rlyle, thanks a lot for reporting. Do you only encounter this issue when running on Windows OS or does it also happen for other OSes? We've seen some similar reports specific to Windows in the past.

**Comment by jbblanton** [comment]
@pgrzesik   I've been getting a very similar error today, with `sls deploy` from Ubuntu 18.04  I have successfully deployed once today, and this is a new issue.  The specific file mentioned in the error has been changing, but the `EMFILE: too many open files` message is consistent.

Only change to my serverless.yml: 

- I added `package: individually: true` 
- I updated from `package: exclude: <list>` to `package: patterns: '!.cache/**', etc.`

**Comment by rlyle** [comment]
> Hello @rlyle, thanks a lot for reporting. Do you only encounter this issue when running on Windows OS or does it also happen for other OSes? We've seen some similar reports specific to Windows in the past.

I've only seen this in windows, when it runs in bitbucket which is a real Linux box, we get no such error so far..

**Comment by nunchukgun** [comment]
@pgrzesik - any chance anyone can take a look at this sometime soon? I'm totally blocked by this issue.

**Comment by pgrzesik** [comment]
Hey @nunchukgun - unfortunately we don't have any ETA or immediate plans to dedicate more time for this bug

**Comment by mnapoli** [comment]
Hey all, we are consolidating issues about this problem into another one and try to make things more actionable.

I'll be closing this in favor of #10944 where we'll try to collect data, share workarounds and ideally identify the root causes.

Let's continue the discussion in #10944
